src/index.html

```diff
@@ -3,11 +3,382 @@
+          </h4>
+          <img
+            class="testimonials__photo"
+            src="./images/client-photo-1.svg"
+            alt="Creative Director Garrett Martin"
+          >
+          <p class="testimonials__name">Garrett Martin</p>
+          <p class="testimonials__position">Creative Director</p>
+        </div>
+      </section>
+
+      <!-- features -->
+
+      <section class="section">
+        <div class="features" id="features">
+          <h3 class="features__title">FEATURES</h3>
+          <div class="features__advantages">
+            <div class="features__advantages-list">
+              <h4>sound & music</h4>
+              <ul>
+                <li>Feel-in-chest Base Power</li>
+                <li>Lossless Digital Audio Transmissio</li>
+                <li>Easy & Stable Stereo Pairing</li>
+                <li>Crisp and Clear High Frequency Sound</li>
+                <li>Streams from Cloud Music and Local Library</li>
+                <li>Auto Music Playback from Last Song Stopped</li>
+              </ul>
+            </div>
+            <div class="features__advantages-list">
+              <h4>connectivity</h4>
+              <ul>
+                <li>Hands Free Wireless Audio</li>
+                <li>Bluetooth 4.0 LE</li>
+                <li>Wi-Fi 2.4 GHz (802.11 b/g/n)</li>
+                <li>Smart Multiroom System Set Up</li>
+                <li>descriptiony Mode with 6.0 Units and above</li>
+                <li>Powerful MESHNET Multi Speaker Network</li>
+              </ul>
+            </div>
+            <div class="features__advantages-list">
+              <h4>app features</h4>
+              <ul>
+                <li>Customize Music Schedule</li>
+                <li>Wake Up with Favorite Songs</li>
+                <li>Home Detection Auto Wake Up</li>
+                <li>Color Wheel & Saturation Change</li>
+              </ul>
+            </div>
+            <div class="features__rectangle"></div>
+          </div>
+        </div>
+      </section>
+
+      <!-- contact -->
+
+      <section class="section">
+        <div class="contact" id="contact">
+          <h3 class="contact__title">Do you have any
+            questions?
+          </h3>
+          <form
+            class="form contact__form"
+            action="#"
+            method="POST"
+            onsubmit="setTimeout(function(){window.location.reload();},10);"
+          >
+            <input
+              type="email"
+              name="email"
+              placeholder="E-mail"
+              class="form__input"
+              id="email"
+              required
+            >
+            <label for="email"></label>
+
+            <textarea
+              type="text"
+              name="message"
+              placeholder="Your message..."
+              class="form__textarea"
+              id="message"
+              required
+            >
+            </textarea>
```

Comment:
Пофіксиш свій баг з полем
```suggestion
            ></textarea>
```